core/services/job/orm.go

```diff
@@ -903,19 +903,19 @@ func (o *orm) loadPipelineRunIDs(jobID *int32, offset, limit int, tx pg.Queryer)
 				// If we're already receiving rows back, then we no longer need an offset
 				offset = 0
 			} else {
-				var skipped int
+				var skipped []int
 				// If no rows were returned, we need to know whether there were any ids skipped
 				//  in this batch due to the offset, and reduce it for the next batch
 				err = tx.Select(&skipped,
 					fmt.Sprintf(
-						`SELECT COUNT(p.id) FROM pipeline_runs AS p %s p.id >= $2 AND p.id <= $3`, filter,
-					),
+						`SELECT COUNT(p.id) FROM pipeline_runs AS p %s p.id >= $1 AND p.id <= $2`, filter,
+					), minID, maxID,
 				)
 				if err != nil {
 					err = errors.Wrap(err, "error loading from pipeline_runs")
 					return
 				}
-				offset -= skipped
+				offset -= skipped[0]
```

Comment:
Why this change to slice?

Reply:
It has to be a slice, because it gets passed as the destination output for `Select()`, which returns a list of rows.  But glad you pointed this out, because it occurs to me now, we could just use `Get()` instead which would be cleaner.  I'll go ahead and change that!